Backend/src/database/s3_database.py

```diff
@@ -46,15 +46,19 @@ def get_course(self, course_code: str, term: str) -> Course | None:
         
         return self.convert_to_course(course_map)
 
-    def get_terms(self) -> List[str]:
+    def get_terms(self) -> dict:
         '''
-        Gets the list of terms in the database.
+        Gets the terms dict in the database.
         '''
-        return list(self.terms_courses_dict.keys())
+        terms_course_copy = self.terms_courses_dict.copy()
+        for term in terms_course_copy:
+            terms_course_copy.get(term, {}).pop("Classes", None)
```

Comment:
If you are just trying to access the values in the dictionary you can loop using terms_course_copy.values()